contracts/ExampleNFT.cdc

```diff
@@ -118,21 +118,21 @@ pub contract ExampleNFT: NonFungibleToken {
         // borrowNFT gets a reference to an NFT in the collection
         // so that the caller can read its metadata and call its methods
         pub fun borrowNFT(id: UInt64): &NonFungibleToken.NFT {
-            return &self.ownedNFTs[id] as &NonFungibleToken.NFT
+            return (&self.ownedNFTs[id] as &NonFungibleToken.NFT?)!
```

Comment:
It is for one of the breaking changes coming with secure Cadence. Did you see the post about it?

https://forum.onflow.org/t/breaking-changes-coming-with-secure-cadence-release/3052

I would highly recommend familiarizing yourself with those because they are important for a lot of projects